pkg/cloud/types.go

```diff
@@ -10,17 +10,44 @@ import (
 
 // InspectOutput is the parsed output from `k6 inspect --execution-requirements`.
 type InspectOutput struct {
-	External struct {
+	External struct { // legacy way of defining the options.cloud
 		Loadimpact struct {
 			Name      string `json:"name"`
 			ProjectID int64  `json:"projectID"`
 		} `json:"loadimpact"`
 	} `json:"ext"`
+	Cloud struct { // actual way of defining the options.cloud
+		Name      string `json:"name"`
+		ProjectID int64  `json:"projectID"`
+	}
```

Comment:
```suggestion
	} `json:"cloud"`
```
Right? This is supposed to be output from `k6 inspect`.